cmd/main.go

```diff
@@ -194,6 +197,7 @@ func main() {
 	}
 
 	port := flag.String("port", "8090", "define port that the service will listen to")
+	httpPort := flag.String("http-port", "8091", "define http port that the service will listen to")
 	flag.Parse()
```

Comment:
:confused: why are these cli args instead of env vars

Reply:
`port` is being set via cli-args, so following along was my first choice. 

However it appears that we'd have to set "command" and pass "http-port" in the operator, so maybe we should move this to env vars?